quadhoral-openmeteo/test_quadhoral_extract.py

```diff
@@ -0,0 +1,128 @@
+'''Contains tests for the extract script in this folder'''
+import pytest
+from unittest.mock import patch, MagicMock
+from quadhoral_extract import get_county_coordinates, convert_to_params, request_weather_data, extract
+from api_error import APIError
+
+class TestGetCountyCoordinates:
+
+    @patch('quadhoral_extract.get_connection')
```

Comment:
Good use of patching for idempotency, avoiding actually interacting with the OpenMeteoAPI or writing to our RDS.